### PR TITLE
Scheduling Time + Performance  + Middleware Server Startup

### LINKED ISSUE #1358: Middleware server - make it survive API restart
- currently the middleware server crashes when API is down 
- could we make the middleware server more robust by automatically re-esatablishing the api subscription in case of short downtime of API?

### PR DESCRIPTION
**Pull request by NilsPur**
closes #1358

### CODE REVIEW

**Review by tpurschke — APPROVED** [review]
@abarz722 can you please check as well?
there seems to be some parallel work going on (e.g. coming soon)

File: roles/middleware/files/FWO.Middleware.Server/Program.cs [hunk 1]
```diff
@@ -26,7 +26,21 @@
...
+{
+    // Repeat first api call in case graphql api is not started yet
+    try
+    {
+        connectedLdaps = apiConnection.SendQueryAsync<List<Ldap>>(AuthQueries.getLdapConnections).Result;
+        break;
+    }
+    catch (Exception ex)
+    {
+        Log.WriteError("Graphql api", "Graphql api unreachable.", ex);
```
Comment (abarz722) [inline roles/middleware/files/FWO.Middleware.Server/Program.cs:40]: Maybe we could put a little wait (maybe increasing with numbers of attempts) here not to blaster the log files?

File: roles/ui/files/FWO.UI/Pages/Reporting/ReportExport.razor [hunk 2]
```diff
@@ -34,9 +34,9 @@
                             </label>
                         </div>
                         <div class="form-check">
-                            <input class="form-check-input" type="checkbox" id="reportExportCsv" @bind="ExportCsv">
+                            <input disabled class="form-check-input" type="checkbox" id="reportExportCsv" @bind="ExportCsv">
                             <label class="form-check-label" for="reportExportCsv">
-                                CSV
+                                CSV (Coming Soon)
```
Comment (abarz722) [inline roles/ui/files/FWO.UI/Pages/Reporting/ReportExport.razor:39]: - Translation necessary. 
- Same handling in Scheduler

Comment (abarz722) [reply roles/ui/files/FWO.UI/Pages/Reporting/ReportExport.razor:39]: will do this in my PR

**Pull request merged**